src/components/SeekGraph/SeekGraph.tsx

```diff
@@ -298,11 +323,13 @@ export class SeekGraph extends TypedEventEmitter<Events> {
                         ]);
                     }
                 }
-                this.challenges[entry.challenge_id] = entry;
+                entry.rank = entry.ranking;
```

Comment:
I think this was right before, see: https://forums.online-go.com/t/broken-graph-on-play-page/50060